adminservice/src/main/java/nu/yona/server/AdminServiceApplication.java

```diff
@@ -17,6 +18,7 @@
 	public static void main(String[] args)
 	{
 		PropertyInitializer.initializePropertiesFromEnvironment();
-		SpringApplication.run(AdminServiceApplication.class, args);
+		ConfigurableApplicationContext context = SpringApplication.run(AdminServiceApplication.class, args);
+		ApplicationStatusLogger.addLoggerForContextClosedEvent(context);
```

Comment:
Do we really need to do it this way? Can't we let `ApplicationStatusLogger` implement `ApplicationEventListener` instead or use [annotations](https://reflectoring.io/spring-boot-application-events-explained/#annotation-driven)?

Reply:
The logger now only logs when 'our' application context is logged. Otherwise, multiple "application closed" messages would be logged for a single close, which would be confusing. I quickly tried whether e.g. the display name would be distinguishing, but it's not: ``Application org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext@6f6745d6 closed``